examples/scroll/lv_example_scroll_1.c

```diff
@@ -1,15 +1,53 @@
 #include "../lv_examples.h"
 #if LV_BUILD_EXAMPLES
 
+lv_obj_t  * panel;
+lv_obj_t  * save_button;
+lv_obj_t  * restore_button;
+int         saved_scroll_x;
+int         saved_scroll_y;
+
+static void scroll_update_cb(lv_event_t * e);
+static void button_event_cb(lv_event_t * e);
+
+static void scroll_update_cb(lv_event_t * e)
+{
+    LV_UNUSED(e);
+
+    LV_LOG("scroll info: x:%3d, y:%3d, top:%3d, bottom:%3d, left:%3d, right:%3d\n",
```

Comment:
```suggestion
    LV_LOG("scroll info: x:%3"LV_PRId32", y:%3"LV_PRId32", top:%3"LV_PRId32", "
           "bottom:%3"LV_PRId32", left:%3"LV_PRId32", right:%3"LV_PRId32"\n",
```

Reply:
@liamHowatt Boy, do I feel silly!  (Where was my head that day?)  \*\*laughing at self\*\*

Good points, all!

Simultaneously you made me look up what `PRId32` (etc.) is for and why they are used.  Thank you for that!  👍 

I re-submitted all the above in 1 commit.